location-search/SearchClient.test.ts

```diff
@@ -0,0 +1,157 @@
+import { Coordinates, Geo } from "@aws-amplify/geo"
+import { mockLocationCoordinate2D, mockTiFLocation } from "@lib/location"
+import AsyncStorage from "@react-native-async-storage/async-storage"
+import awsconfig from "../src/aws-exports"
+import { asyncStorageSaveRecentLocation } from "./RecentsStorage"
+import {
+  awsPlaceToTifLocation,
+  searchRecentLocations,
+  searchWithRecentAnnotations
+} from "./SearchClient"
+import { mockLocationSearchFunction } from "./mocks"
+import { LocationsSearchQuery } from "./models"
+
+Geo.configure(awsconfig)
```

Comment:
We can remove this part since you mention it’s causing issues. I’ll test it myself at a later date